crates/net/src/eventsource/mod.rs

```diff
@@ -11,7 +11,7 @@ use std::fmt;
 ///
 /// See [`EventSource.readyState` on MDN](https://developer.mozilla.org/en-US/docs/Web/API/EventSource/readyState)
 /// to learn more.
-#[derive(Copy, Clone, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
I'm not on the `gloo` team, but would it make sense to add `PartialEq, Eq, PartialOrd, Ord` behind a feature flag and `Hash` behind another? I think people care about code bloat when it comes to wasm (depending on where the code is being run).